Example-Crypto-REXX/RXCRYPT1.rexx

```diff
@@ -0,0 +1,2399 @@
+/*        EX 'HWI.HWIREST.REXX(RXCRYPT1))'                             */
+/*            '-D HWI.CRYPTO -I'                                       */
+/*            data set is HWI.CRYPTO, run out of ISV REXX,             */
+/*            if successful will create HWI.CRYPTO(LOCAL)              */
+/*                                                                     */
+/*        EX 'HWI.HWIREST.REXX(RXCRYPT1))'                             */
+/*            '-D HWI.CRYPTO -S not-activated'                         */
+/*            data set is HWI.CRYPTO, run from TSO REXX,               */
+/*            query crypto attributes for not-activated LPARs          */
+/*            if successful will create HWI.CRYPTO(LOCAL)              */
+/*                                                                     */
+/* DEPENDENCIES:                                                       */
+/*     z16 - The crypto properties are supported on z16 processors    */
+/*           and above.                                                */
+/*                                                                     */
+/* NOTES:                                                              */
+/*     No recovery logic has been supplied in this sample.             */
+/*                                                                     */
+/* REFERENCE:                                                          */
+/*     See the z/OS MVS Programming: Callable Services for             */
+/*     High-Level Languages publication for more information           */
+/*     regarding the usage of HWIREST and JSON Parser APIs.            */
+/*                                                                     */
+/*                                                                     */
+/* END OF SPECIFICATIONS  * * * * * * * * * * * * * * * * * * * * * * **/
+
+MACLIB_DATASET = 'SYS1.MACLIB'
+
+TEMP_DATASET = '' /* required input via -D */
+MEMBER_NAME = 'LOCAL' /* default CPC */
+
+TRUE = 1
+FALSE = 0
+
+hwiHostRC = 0     /* HWIHOST rc */
+HWIHOST_ON = FALSE
+PARSER_INIT = FALSE
+ISVREXX = FALSE  /* default to TSO/E, enable via -I */
+VERBOSE = FALSE  /* JSON parser specific, enabled via -V */
+localCPC = TRUE  /* default to lpars on the LOCAL CPC, change via -C */
+
+/* Properties constants */
+cryptoauth = 'crypto-activity-cpu-counter-authorization-control'
+assignedDomainsProp = 'assigned-crypto-domains'
+assignedCryptosProp = 'assigned-cryptos'
+
+/*********************************/
+/* Get program args              */
+/*********************************/
+parse arg argString
+if GetArgs(argString) <> 0 then
+  exit -1
+
+/**************************************/
+/* Before proceeding, ensure the      */
+/* data set specified actually exists */
+/**************************************/
+if VerifyDataSet(TEMP_DATASET) <> 0 then
+  exit fatalErrorAndCleanup('** specified data set does not exist **')
```

Comment:
You may want to add what non-existing data set the user attempted to utilize

Reply:
Fixed.